components/sdmmc/sdmmc_cmd.c

```diff
@@ -25,6 +25,7 @@
 #include "sdmmc_cmd.h"
 #include "sys/param.h"
 #include "soc/soc_memory_layout.h"
+#include "soc/sdmmc_struct.h"
```

Comment:
Ideally the protocol layer should not know about host registers. Can this be removed?

Reply:
removed